tcms/testcases/tests/test_admin.py

```diff
@@ -0,0 +1,25 @@
+# -*- coding: utf-8 -*-
+# pylint: disable=invalid-name
+from django.urls import reverse
+
+from tcms.tests import LoggedInTestCase
+from tcms.tests.factories import TestCaseFactory
+from tcms.utils.permissions import initiate_user_with_default_setups
+
+
+class TestTestCaseAdmin(LoggedInTestCase):
+
+    @classmethod
+    def setUpTestData(cls):
+        super().setUpTestData()
+        initiate_user_with_default_setups(cls.tester)
+        cls.test_case = TestCaseFactory()
+
+    def test_users_can_not_add_testcase(self):
```

Comment:
Test names should follow the actual behavior! Analyze what is the actual behavior and you fill figure out the name,

Reply:
ping